plaid/plaid.go

```diff
@@ -1,373 +1,147 @@
-// Package plaid implements a Go client for the Plaid API (https://plaid.com/docs)
 package plaid
 
 import (
+	"bytes"
 	"encoding/json"
 	"errors"
 	"io"
 	"io/ioutil"
 	"net/http"
-)
+	"path/filepath"
+	"strings"
 
-// NewClient instantiates a Client associated with a client id, secret and environment.
-// See https://plaid.com/docs/api/#gaining-access.
-func NewClient(clientID, secret string, environment environmentURL) *Client {
-	return &Client{clientID, secret, environment, &http.Client{}}
-}
+	"github.com/BurntSushi/toml"
+)
 
-// Same as above but with additional parameter to pass http.Client. This is required
-// if you want to run the code on Google AppEngine which prohibits use of http.DefaultClient
-func NewCustomClient(clientID, secret string, environment environmentURL, httpClient *http.Client) *Client {
-	return &Client{clientID, secret, environment, httpClient}
-}
+// LatestPlaidAPIVersion holds the latest version of the Plaid API
+const LatestPlaidAPIVersion = "2018-05-22"
```

Comment:
i think this should just be `LatestAPIVersion`

Reply:
⚡️